app/apps/cases/migrations/0027_auto_20210120_1346.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 3.1.5 on 2021-01-20 12:46
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cases", "0026_case_camunda_id"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="CaseTeam",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="CaseReason",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                (
+                    "case_team",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.PROTECT,
+                        related_name="case_reasons",
+                        to="cases.caseteam",
+                    ),
+                ),
+            ],
+        ),
+        migrations.AddField(
+            model_name="case",
+            name="case_reason",
+            field=models.ForeignKey(
+                null=True,
+                on_delete=django.db.models.deletion.PROTECT,
+                to="cases.casereason",
+            ),
+        ),
+        migrations.AddField(
+            model_name="case",
+            name="case_team",
+            field=models.ForeignKey(
+                null=True,
```

Comment:
Lijkt mij beter deze verplicht te maken en met een manual migration een team aan te maken. Kan na deze PR wel

Reply:
Done. Heb het opgelost door een default Team en Reason aan te maken. Die zijn instelbaar via de enviroment variables, zodat het niet te hardcoded is allemaal. Er vindt een data migratie plaats die er voor zorgt dat ze toegevoegd worden.

Reply:
Done. Heb ze verplicht en met een datamigratie defaults toegevoegd. (Deze Team en Reason zijn ook nog eens configureerbaar via de enviroment variables, dus daardoor voorkomen we dat het te hardcoded wordt)

Reply:
Nice!